drivers/media/platform/bcm2835/bcm2835-unicam.c

```diff
@@ -2155,11 +2170,31 @@ static int unicam_probe(struct platform_device *pdev)
 		return -EINVAL;
 	}
 
+	unicam->mdev.dev = &pdev->dev;
+	strscpy(unicam->mdev.model, UNICAM_MODULE_NAME,
+			sizeof(unicam->mdev.model));
```

Comment:
Our old friend checkpatch whinges "Alignment should match open parenthesis"

Reply:
Argh, I have a commit hook that should have caught that (all my commits should get run through checkpatch at commit time and rebase time)...
Not sure why it didn't.
I'll fix up.

Reply:
ah - checkpatch runs clean - but checkpatch --strict doesn't :) I'll update my hooks accordingly.

Reply:
:-) I know the frustration.
I have the same sort of hook (hence I found this on cherry-pick), and generally swear at it.